config/rbac/leader_election_role.yaml

```diff
@@ -29,3 +29,14 @@ rules:
   - events
   verbs:
   - create
+- apiGroups:
+    - ""
+  resources:
+    - leases
+  verbs:
```

Comment:
Since we are replacing configmaps with leases, I was using the same verbs that configmaps already has.


```
  resources:
  - configmaps
  verbs:
  - get
  - list
  - watch
  - create
  - update
  - patch
```

I wasn't sure what is required for leader election. Is list, and watch required? I wasn't certain and so used the ones that we already used for configmap.